mteb/tasks/Classification/ita/ItaHateClassification.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItaHateClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="ItaHateClassification",
+        dataset={
+            "path": "Paul/hatecheck-italian",
+            "revision": "21e3d5c827cb60619a89988b24979850a7af85a5",
+        },
+        description="""Hate speech detection dataset with binary
+                       (hateful vs non-hateful) labels. Includes 25+ distinct types of hate
+                       and challenging non-hate. Multilingual datase released as 10 unilingual models
+                     """,
+        reference="https://aclanthology.org/2022.woah-1.15/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
```

Comment:
If you make it multilingual this will turn into a mapping:
```suggestion
        eval_langs={"language_split_name_on_hf": ["ita-Latn"], ... }
```

Reply:
related to this and the previous comment, while this is originally a multilingual dataset, it is released on HF as a set of monolingual datasets. Should I combine them? I was originally thinking of just adding them as separate monolingual datasets?